### ISSUE TITLE
[FEATURE] - Add search function

### ISSUE THREAD

**Issue by leandercai** [alefragnani/vscode-bookmarks#417]
Thank you for offering such useful plugin. For big project, many bookmarks maybe are marked. It is very useful if we can search saved bookmarks.

**Comment by alefragnani** [comment]
Hi @leandercai ,

You already can do that. Just use the `Bookmarks: List from All Files` command. You will be able to search/filter by _line content_, _line number_ and _bookmark label_.

Hope this helps

**Comment by leandercai** [comment]
Hi @alefragnani 
Yes, I found the command ` Bookmarks: List from All Files`. But we should choose a folder first. For a multi-folder project, we don't know the exact folder when searching. I guess perhaps it is related to the multi `bookmarks.json` files. Would you pls do some changes to make it more convenient?

Thanks.

**Comment by alefragnani** [comment]
So your request is specially related to **multi-root workspace** scenarios?

I decided to use that approach because it is how I noticed other VS Code native commands also do. It was a matter of following the same UI, but I understand your request.

Well, one way to accomplish that is to add a `setting`, so you could decide how the command would work.

Hope this helps

**Comment by leandercai** [comment]
Yes, in most case, we search a bookmark in the whole project. And I think it is more convenient if there is a search box in the top of bookmarks UI.

Thanks a lot.

**Comment by alefragnani** [comment]
Unfortunately, there is no way to add a **search box** on the top of the Bookmarks UI, because VS Code doesn't provide an API for that. So it will remains using the same **list**, with no `folder` selection step.

**Comment by leandercai** [comment]
If so, your solution can work as well. 

I have an enhencement on my feature request. For many bookmarks, we didn't add label. That means we just use the current line acting as label. Would you please add a option so that we can search both label and current line(for unlabeled bookmarks)? 

Thanks.

**Comment by alefragnani** [comment]
There is no need to add a new setting. The current `List from All Files` command already allows you to filter/search bookmarks by _line content_, _line/column number_ or _label_. 

So, no matter if you use Label or not, you will be able to find any bookmarked line. Just type any of these informations, and the bookmarks will be filtered/found

BTW, I don't use labels myself.

**Comment by leandercai** [comment]
Sorry! It's supported already。I have chosen another folder in the project wrongly. So it is very useful to search labels in the total project。Looking forward to the new feature.

Thanks.
